btr-bagit-profile.json

```diff
@@ -2,12 +2,12 @@
     "BagIt-Profile-Info": {
         "Source-Organization": "Beyond the Repository Bagit Profile Group",
         "External-Description": "Bagit Profile for Consistent Deposit to Distributed Digital Preservation Services",
-        "Version": "0.1",
-        "BagIt-Profile-Identifier": "http://????/???-v0.1.json",
+        "Version": "1.0",
+        "BagIt-Profile-Identifier": "https://github.com/dpscollaborative/btr_bagit_profile/releases/download/1.0/btr-bagit-profile.json",
         "BagIt-Profile-Version": "1.3.0",
-        "Contact-Name": "Hope Fully Apersonsname",
-        "Contact-Phone": "(we probably don't want this)",
-        "Contact-Email": "fake-email@example.com"
+        "Contact-Name": "",
+        "Contact-Phone": "",
+        "Contact-Email": ""
```

Comment:
Is there a benefit to retaining these `Contact-*` parameters with no value? I had assumed that we'd just pull them out entirely.

Reply:
My mistake. I thought the Contact fields had to be present if empty. Spec says they're optional, so I removed them.